rackcity/views/it_model_views.py

```diff
@@ -50,25 +54,46 @@ def model_add(request):
     Add a new model
     """
     data = JSONParser().parse(request)
-    failure_message = ""
     if 'id' in data:
-        failure_message = failure_message + "Don't include id when adding a model. "
+        return JsonResponse(
+            {
+                "failure_message":
+                    Status.CREATE_ERROR.value + GenericFailure.UNKNOWN.value,
+                "errors": "Don't include 'id' when creating a model"
+            },
+            status=HTTPStatus.BAD_REQUEST
+        )
```

Comment:
Updating all endpoints to fail early with explicit error messages based on previous convos